### PR TITLE
[Update] Ignore authors when compiling the correlations, also tidy logging

### PR DESCRIPTION
**Pull request by railsagainstignorance**
# Description

Ignore authors when compiling the correlations, also tidy logging.

Authors introduce false correlations between entities, especially people,  in the correlation graph.

There was lots of spurious commented-out logging, and assorted use of console.log. Most of the live console.log have been changed to debug(), leaving just a couple of console.log to write useful info to logs.

# Implementation

Made use of the existing ignoreEntities object. Any authors:* entities were rewritten into people:* entities and pushed into ignoreEntities.

ignoreEntitiesList is also exposed via the /summary endpoint.

# Screenshot
N/A

# .env changes
N/A

# Additional requirements

Needed a much-belated update to get latest version of s30.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -84,11 +84,39 @@ if (process.env.BYPASS_TOKEN !== 'true') {
 	app.use(validateRequest);
 }
 
+function sortIsland( island ){
+  // sort all the entities by count and then alphabetically,
+  // assuming island = { e1: num1, e2: num2, ...}
+  const entities = Object.keys(island).sort( (a,b) => {
+    if(island[a] < island[b]){ return +1; }
```
Comment (Lily2point0) [inline index.js]: what if `island[a] === island[b]` ?

File: index.js [hunk 2]
```diff
@@ -84,11 +84,39 @@ if (process.env.BYPASS_TOKEN !== 'true') {
...
+  }
+}
+
 app.get('/', (req, res) => {
-	// res.sendFile(path.join(__dirname + '/static/index.html'));
+  // find the biggest island
   const islands = correlate.allIslands();
-  let island = (islands.length > 0)? islands[0] : [ {'entity1': true, 'entity2' : true}];
-  const entities = Object.keys(island);
+  let island = (islands.length > 0)? islands[0] : [ {'entity1': 1, 'entity2' : 1}];
```
Comment (Lily2point0) [inline index.js]: is `island getting re-assigned later or can it be a `const`?

File: index.js [hunk 3]
```diff
@@ -84,11 +84,39 @@ if (process.env.BYPASS_TOKEN !== 'true') {
...
+  const entities = sortIsland( island ).entities;
+
+  console.log( `DEBUG: /: island=${JSON.stringify(island, null, 2)},
```
Comment (Lily2point0) [inline index.js]: How do we do logging in the rest of the project? Isn't it with `debug`?

File: index.js (hunk 1 above)
Comment (railsagainstignorance) [reply index.js]: Then sort by count initially, then alphabetically if the counts are equal, in the else {}, which itself has an else { return 0; } , which, thinking about it, should never happen, given that they should all be unique keys strings, but anyway.

File: index.js (hunk 2 above)
Comment (railsagainstignorance) [reply index.js]: now const

File: index.js (hunk 3 above)
Comment (railsagainstignorance) [reply index.js]: yes, now debug

**Pull request merged**